src/Components/CreateImageEntry.jsx

```diff
@@ -136,13 +223,12 @@ class CreateImageEntry extends Component {
 
           <Sidebar.Pusher dimmed={this.state.messageVisible}>
             <Container id="upload-post-wrapper">
-              <Header id="upload-post-header" as='h3'>Add a photo</Header>
               <ImageUploader
                 buttonText={
                   <div>
                     <p id="add-photo-headline">Add Image</p>
                     <Icon id="add-photo-icon" name="image outline" size="huge"></Icon>
-                    <p id="add-photo-label">Maximum image file size: 5 MB, Accepted image types: JPG</p>
+                    <p id="add-photo-label">Maximum image file size: 5 MB, Accepted image types: JPG/JPEG/PNG/GIF</p>
```

Comment:
I would remove this "Accepted image types: JPG/JPEG/PNG/GIF" altogether. Most users will choose an image. If they don't, you can raise an error and tell the user about the image formats at that point. Less is more ;-)

Reply:
Makes sense, and also gives us more space for the popup content - which is a welcomed action :) Changed now.